Assets/Scripts/SaveSystem/GameContext.cs

```diff
@@ -4,10 +4,13 @@ namespace SpaceGame.SaveSystem
 {
     public static class GameContext
     {
+        public static int count;
         public static GameData CurrentGameData { get; set; }
 
         public static PlayerData PlayerData1 => CurrentGameData.PlayersData[0];
 
         public static PlayerData PlayerData2 => CurrentGameData.PlayersData[1];
+
+        public static EnemyData EnemysData => CurrentGameData.EnemiesData[count];
```

Comment:
This entry returns the single date of the enemy.
And there can be any number of them.
Here it makes sense to use a method in which you can pass a parameter index

Reply:
+?